3p/iframe-messaging-client.js

```diff
@@ -227,4 +268,11 @@ export class IframeMessagingClient {
     // Learned from https://github.com/dtapuska/useractivation
     return this.hostWindow_ && this.hostWindow_.postMessage.length == 1;
   }
+
+  /**
+   * @param {boolean} mode
+   */
+  setBroadcastMode(mode) {
```

Comment:
do we need this method? seems unnecessarily exposing such  an implementation details about how we find the host window.

Reply:
Deleted.